css/index.css

```diff
@@ -143,4 +299,79 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+    justify-content: center;
+    margin-left:40px;
+}
+
+/* ABOUT PAGE */
+
+
+
+
+.top-half .header-img {
+display:flex;
+flex-direction: row;
+justify-content: center;
+margin-left: 310px;
+}
+
+.header-text{
+
+    display:flex;
+    width: 50%;
+    justify-content: center;
+    flex-direction:column;
+    margin-left: 310px;
+    margin-top: 20px;
+}
+
+.main-top{
+    display:flex;
+    width:50%;
+    justify-content: column;
+    flex-wrap: nowrap;
+    align-items: flex-end;
+    margin-left: 310px;
+    
+}
+.main-top .strategy {
+    text-align: left;
+    margin-right: 50px;
+    margin-bottom: 50px;
+}
+
+.main-top .How-We-Work {
+    text-align: left;
+    margin-right: 50px;
+    margin-bottom: 50px;
+}
+
+.main-bottom{
+    display:flex;
+    width:50%;
+    justify-content: column;
+    flex-wrap: nowrap;
+    align-items: flex-end;
+    margin-left: 310px;
+}
+
```

Comment:
Great job in using the appropriate Flexbox properties to maintain uniformity in both the top and bottom cards. Think about how you could style all four with one instance of code. Perhaps something like a `main-card` class applied on both top and bottom to snatch them all at one since the styling is repeated.